fixed-hash/Cargo.toml

```diff
@@ -19,6 +19,7 @@ quickcheck = { version = "0.9.0", optional = true }
 rand = { version = "0.7.2", optional = true, default-features = false }
 rustc-hex = { version = "2.0.1", optional = true, default-features = false }
 static_assertions = "1.0.0"
+arbitrary = { version = "0.3", optional = true }
```

Comment:
Curious about the choice of version here, why not use `0.4.2`?

Reply:
Last I checked, [libfuzzer](https://docs.rs/libfuzzer-sys/0.3.2/libfuzzer_sys/index.html) who is the one of main users of `arbitrary` was pointing to `0.3.0` but looks live they've updated to `0.4.1` now so I'll bump the version to `0.4`